third_party/move/move-compiler-v2/tests/checking/inlining/private_call_3.move

```diff
@@ -0,0 +1,102 @@
+module 0x42::m {
```

Comment:
I think we also need a test case for exposing constants via inlining. Consider the following transactional test, as an example.

```
//# publish
module 0xc0::one {
    const CC: u64 = 1;

    public inline fun blah(): u64 {
        CC
    }
}

//# publish
module 0xc0::two {
    public fun foo(): u64 {
        0xc0::one::blah()
    }
}

//# run 0xc0::two::foo
```

v1 gives an error saying bytecode generation failed, v2 returns 1.

We should likely fix the former to be a nicer error message than an ICE, but also make sure we deliver the same semantics.